src/lumigo_opentelemetry/__init__.py

```diff
@@ -94,20 +94,35 @@ def init() -> Dict[str, Any]:
     )
 
     from opentelemetry import trace
+    from opentelemetry import _logs
+    from opentelemetry.instrumentation.logging import LoggingInstrumentor
     from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
     from opentelemetry.sdk.trace import SpanLimits, TracerProvider
-
+    from opentelemetry.sdk._logs import LoggerProvider, LoggingHandler
+    from opentelemetry.sdk._logs.export import BatchLogRecordProcessor
+    from opentelemetry.exporter.otlp.proto.http._log_exporter import OTLPLogExporter
     from lumigo_opentelemetry.resources.span_processor import LumigoSpanProcessor
 
     DEFAULT_LUMIGO_ENDPOINT = (
         "https://ga-otlp.lumigo-tracer-edge.golumigo.com/v1/traces"
     )
+    DEFAULT_LUMIGO_LOGS_ENDPOINT = (
+        "https://ga-otlp.lumigo-tracer-edge.golumigo.com/v1/logs"
+    )
     DEFAULT_DEPENDENCIES_ENDPOINT = (
         "https://ga-otlp.lumigo-tracer-edge.golumigo.com/v1/dependencies"
     )
-    lumigo_endpoint = os.getenv("LUMIGO_ENDPOINT", DEFAULT_LUMIGO_ENDPOINT)
+    lumigo_traces_endpoint = os.getenv("LUMIGO_ENDPOINT", DEFAULT_LUMIGO_ENDPOINT)
+    lumigo_logs_endpoint = os.getenv(
+        "LUMIGO_LOGS_ENDPOINT", DEFAULT_LUMIGO_LOGS_ENDPOINT
+    )
     lumigo_token = os.getenv("LUMIGO_TRACER_TOKEN")
-    lumigo_report_dependencies = os.getenv("LUMIGO_REPORT_DEPENDENCIES", "true").lower()
+    lumigo_report_dependencies = (
+        os.getenv("LUMIGO_REPORT_DEPENDENCIES", "true").lower() == "true"
+    )
+    logging_enabled = os.getenv("LUMIGO_LOGS_ENABLED", "").lower() == "true"
+    spandump_file = os.getenv("LUMIGO_DEBUG_SPANDUMP")
+    logdump_file = os.getenv("LUMIGO_DEBUG_LOGDUMP")
```

Comment:
any new env var you introduced -> please add to the `README.md`

Reply:
also we need to consider adding a proper paragraph in the README about this feature and how to use and consume it in Lumigo

Reply:
Yeah definitely, I was just wondering if it's better to do that once the secret-scrubbing is in place and leave it "undocumented" at the moment